src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx

```diff
@@ -0,0 +1,46 @@
+import { CalendarOutlined } from "@ant-design/icons"
+import { PHIcon } from "@ph/ui"
+import React from "react"
+
+import { formatDateToLongString, formatDateToTimeSlot } from "@/utils/dateTime"
+
+import { LIInterviewMode, LIInterviewModeLabel } from "../MobileCandidateSelectInterviewTime/constants"
+import { useCandidateConfirmation } from "./hooks/useCandidateConfirmation"
+import { MobileCandidateConfirmationInfo } from "./MobileCandidateConfirmationInfo"
+
+export const MobileCandidateConfirmation = () => {
+  const { interviewData } = useCandidateConfirmation()
+  const { jobRequisitionName, interviewMode, interviewAddress, timezone, interviewTime, interviewLink } = interviewData
+  const interviewDate = formatDateToLongString(new Date(interviewTime.start), timezone)
+  const interviewTimeSlot = formatDateToTimeSlot(interviewTime, timezone)
+
+  return (
+    <article className="candidateConfirmationWrapper h-screen overflow-y-auto bg-gray-100 p-5">
+      <div className="h-auto w-full max-w-sm rounded-[20px] bg-white p-5 shadow-card">
+        <div className="mb-6 text-center">
+          <PHIcon size={140} name="interviewConfirmation" />
+          <h2 className="mb-2 mt-4 text-lg font-semibold text-black">
+            We look forward to seeing you on {`${interviewDate} ${interviewTimeSlot}`}!
+          </h2>
```

Comment:
Can we do something like this? 
```suggestion
          <h2 className="mb-2 mt-4 text-lg font-semibold text-black">
            We look forward to seeing you on <time>{`${interviewDate} ${interviewTimeSlot}`}</time>!
          </h2>
```
The suggestion I made here is not complete, you need to provide the time value to it https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time